packages/rocketchat-ui/client/views/app/popover.html

```diff
@@ -9,7 +9,7 @@ <h3 class="rc-popover__title">{{group.title}}</h3>
 						{{/if}}
 						<ul class="rc-popover__list">
 							{{#each item in group.items}}
-							<li class="rc-popover__item rc-popover__item--{{item.class}}" data-type={{item.type}} data-id={{item.id.toLowerCase}} data-href={{item.href}} data-sidenav={{item.sideNav}}>
+							<li class="rc-popover__item rc-popover__item--{{item.modifier}}" data-type={{item.type}} data-id={{item.id.toLowerCase}} data-href={{item.href}} data-sidenav={{item.sideNav}}>
```

Comment:
whats happen when the modifier is null?

Reply:
It will create an `rc-popover__item--` class. Is it a problem?

Reply:
yes.